### PR TITLE
ngx-build: renamed ssl_pending_session.patch to ssl_sess_cb_yield.pat…

### PR DESCRIPTION
**Pull request by spacewander**
…ch since Nginx 1.15.8.

### CODE REVIEW

File: ngx-build [hunk 1]
```diff
@@ -314,7 +314,13 @@ sub apply_patches {
     chdir ".." or die "cannot switch to ..\n";
 
     chdir "nginx-$version" or die "cannot switch to nginx-$version\n";
-    shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_pending_session.patch");
+    unless ($ver ge '001015008') {
+        shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_sess_cb_yield.patch");
+
+    } else {
```
Comment (agentzh) [inline ngx-build]: It's confusing to use `unless` with `else`. My brain is twisted. Please switch to `if` `else` instead.

File: ngx-build [hunk 2]
```diff
@@ -314,7 +314,13 @@ sub apply_patches {
     chdir ".." or die "cannot switch to ..\n";
 
     chdir "nginx-$version" or die "cannot switch to nginx-$version\n";
-    shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_pending_session.patch");
+    if ($ver ge '001015008') {
+        shell("patch -p1 < $root/../openresty/patches/nginx-$version-ssl_pending_session.patch");
```
Comment (thibaultcha) [inline ngx-build]: This branch should be applying `ssl_sess_cb_yield.patch`, and not `ssl_pending_session.patch`.

Comment (thibaultcha) [reply ngx-build]: Pushed the fix to the PR

**Comment by thibaultcha** [comment]
Merged!

**Pull request closed without merging**